resources/css/leaflet/Leaflet.Fullscreen.css

```diff
@@ -0,0 +1,40 @@
+.leaflet-control-fullscreen a {
+  /*background:#fff url(images/fullscreen.png) no-repeat 0 0;
```

Comment:
Unused Code can be deleted 👍

Reply:
Mixing does this stuff while running on production, removing comments or empty rules, doesn't matter but having in mind.

I found the only exception is when comments has this format:
```css
/*!
 You-shall-not-pass
*/
```
.. Then webpack mix will not ~~pass~~ work.

Reply:
Good to know